src/pages/concept-registration-page/form-term/form-term.component.tsx

```diff
@@ -17,7 +17,8 @@ interface FormProps {
 const config = {
   mapPropsToValues: ({ concept }: FormProps) => ({
     anbefaltTerm: _.get(concept, 'anbefaltTerm') || '',
-    definisjon: _.get(concept, 'definisjon') || ''
+    definisjon: _.get(concept, 'definisjon') || '',
```

Comment:
Vet du hvordan havnet norsk terminologi i modell? Jeg tror ikke at det er bra ide, men det er vanskelig å endre det nå.

Reply:
er på norsk i speccen også... tror vi i denne omgang bare må håndtere det som teknisk gjeld dersom vi ønsker å endre på det senere... Er helt enig med deg at vi bør ha som standard at alt er på engelsk.

Reply:
+1 for spec på engelsk.